### PR TITLE
refactor: refactor drivers readability

### PR DESCRIPTION
**Pull request by nan-wang**
minor changes to improve the readability

### CODE REVIEW

File: jina/drivers/__init__.py [hunk 1]
```diff
@@ -698,9 +702,9 @@ def attach(
             decor_bindings = find_request_binding(self.exec.__class__)
             if req_type:
                 canonic_name = _canonical_request_name(req_type)
-            if req_type and canonic_name in decor_bindings:
-                self._method_name = decor_bindings[canonic_name]
-            elif 'default' in decor_bindings:
+                if canonic_name in decor_bindings:
+                    self._method_name = decor_bindings[canonic_name]
+            if 'default' in decor_bindings:
```
Comment (JoanFM) [inline jina/drivers/__init__.py:707]: this is an actual functionality change

Comment (nan-wang) [reply jina/drivers/__init__.py:707]: L707?

File: jina/drivers/__init__.py [hunk 2]
```diff
@@ -698,8 +702,10 @@ def attach(
             decor_bindings = find_request_binding(self.exec.__class__)
             if req_type:
                 canonic_name = _canonical_request_name(req_type)
-            if req_type and canonic_name in decor_bindings:
-                self._method_name = decor_bindings[canonic_name]
+                if canonic_name in decor_bindings:
+                    self._method_name = decor_bindings[canonic_name]
+                elif 'default' in decor_bindings:
+                    self._method_name = decor_bindings['default']
```
Comment (nan-wang) [inline jina/drivers/__init__.py:708]: @JoanFM Now it is consistent with the old codes.

**Review by JoanFM — APPROVED** [review]

**Comment by nan-wang** [comment]
The PR is messed up due to the mixed usage of `rebase` and `merge`. The changes are done in #2357

**Pull request closed without merging**